### PR TITLE
Bug 1489709: Use create project popup rather than the create-project page

### LINKED ISSUE #2029: Should warn when secret keys are not valid env var names
In the "Add to Application" dialog, we should warn when some secret keys aren't valid environment variable names. These keys will be skipped when adding env variables.

cc @beanh66

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
https://bugzilla.redhat.com/show_bug.cgi?id=1489709

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
[test]

File: app/views/projects.html [hunk 1]
```diff
@@ -170,7 +170,12 @@ <h2 class="h1">
                 OpenShift helps you quickly develop, host, and scale applications.<br>
                 <span ng-if="canCreate">Create a project for your application.</span>
               </p>
-              <a ng-if="canCreate" href="create-project" class="btn btn-lg btn-primary">Create Project</a>
+              <div>
+                <button ng-click="createProject($event)" class="btn btn-md btn-primary">
```
Comment (spadgett) [inline app/views/projects.html]: I don't think there is a `btn-md` (unless it's something Patternfly has added to Bootstrap).

https://getbootstrap.com/docs/3.3/css/#buttons-sizes

File: app/views/projects.html [hunk 2]
```diff
@@ -17,13 +20,10 @@ <h2 class="text-center">Loading...</h2>
                   <h1>My Projects</h1>
                   <div class="projects-options">
                     <div class="projects-add" ng-if="canCreate">
-                      <button ng-click="createProject()" class="btn btn-md btn-primary">
+                      <button ng-click="createProject($event)" class="btn btn-md btn-primary">
```
Comment (spadgett) [inline app/views/projects.html]: Same comment on `btn-md`, although I see that was already there.

**Comment by spadgett** [comment]
> Origin Web Console Test Results: FAILURE >
> (https://ci.openshift.redhat.com/jenkins/job/test_pull_request_origin_web_console/148/) (Base Commit: db9abb5) (PR Branch Commit: da39515)

Flake #2037

File: app/views/projects.html (hunk 1 above)
Comment (jeff-phillips-18) [reply app/views/projects.html]: Changed it back to btn-lg

File: app/views/projects.html (hunk 2 above)
Comment (jeff-phillips-18) [reply app/views/projects.html]: removed the btn-md

**Comment by spadgett** [comment]
[merge][severity: bug]

**Comment by spadgett** [comment]
[merge][severity: bug]

**Comment by spadgett** [comment]
flake #2016

[merge][severity: bug]

**Pull request merged**